27. LeetCode Problems/Maximal Network Rank.cpp.cpp

```diff
@@ -0,0 +1,87 @@
+/*
+
+Leetcode - Maximal Network Rank
```

Comment:
Check file name .cpp.cpp ? Update this.

Reply:
oops. I've corrected it.

Reply:
File name contain spaces. Please follow naming guideline specified in readme. No spaces allowed.